app/controllers/application_controller.rb

```diff
@@ -753,7 +753,7 @@ def get_reports_menu(group = current_group, tree_type = "reports", mode = "menu"
     user = current_user
     @sb[:grp_title] = reports_group_title
     @data = []
-    if (!group.settings || !group.settings[:report_menus] || group.settings[:report_menus].blank?) || mode == "default"
+    if (!group.settings || group.settings[:report_menus].blank?) || mode == "default"
```

Comment:
I imagine this is perhaps not obvious by looking up the pr as whole. I try to commits as a story, so it should be easier to comprehend my prs reading commit by commit. Thanks!